index.js

```diff
@@ -0,0 +1,128 @@
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p><ul>`;
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+const loadTrips = () => {
+  reportStatus('Loading trips . . .')
+
+  const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
```

Comment:
I'd recommend making this a constant outside of this function so it can be reused elsewhere